front/ticket.form.php

```diff
@@ -86,7 +86,8 @@
             // Sanitize before merging with $_POST['comment'] which is already sanitized
             'content'    => Sanitizer::sanitize(
                 '<p><i>' . sprintf(__('Escalation to the group %s.', 'escalade'), Sanitizer::unsanitize($group->getName())) . '</i></p><hr />'
-            ) . $_POST['comment']
+            ) . $_POST['comment'],
+            '_do_not_compute_takeintoaccount' => true
```

Comment:
A dedicated option should be creating from config page to compute or not TTO after escalation, because actually TTO is computed (and the community never complains).

Reply:
Yes, an option that will have to take into account that it will only work with a certain version of GLPI (probably 10.0.18).